scripts/ayp.coffee

```diff
@@ -312,13 +312,17 @@ class AYPStrip
 
     @loadAvatars names, (err, avatars) =>
       return cb(err, null) if err
+      scrambleLines = []
 
       if names.length == 1
         # The only person speaking is centered in the frame
         char = avatars[names[0]]
         left = (frame.width / 2) - (char.width / 2)
         top = (frame.height - char.height)
         @compositeImage frame, char, left, top
+
+        isKenny = char.matchchar.match('default') == null
+        scrambleLines = line.concat(isKenny) for line in lines
```

Comment:
Please return a copy here instead of mutating using `.concat`, should be as easy as `[line..., isKenny]`

And if you use a right-hand loop, please parenthesize it.